src/Abp.HtmlSanitizer/HtmlSanitizer/HtmlSanitizerBuilder.cs

```diff
@@ -27,7 +27,7 @@ public static IHtmlSanitizerConfiguration AddSelector<T>(this IHtmlSanitizerConf
 
                 if (!typeof(T).IsAssignableFrom(type) || !type.IsClass) return false;
 
-                var instance = (T) FormatterServices.GetUninitializedObject(type);
+                var instance = (T) default;
```

Comment:
Hi @ymg2006,

When I tested it here, I saw that it wasn't working. Since the method is deprecated, I can recommend the following solution. 

`var instance = (T) Activator.CreateInstance(type, nonPublic: true);`